doorman/application.py

```diff
@@ -1,8 +1,9 @@
 # -*- coding: utf-8 -*-
 import os
+import sys
 
-from flask import Flask, render_template
-
+from flask import Flask, render_template, request, redirect
```

Comment:
Importing `request` and `redirect` seem unnecessary here... Artifacts from earlier commits?